### DISCUSSION TITLE
Help! Account flagged for apparently trying to open multiple accounts!

### DISCUSSION THREAD

**Discussion by kittyg80** [community/community#46415] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, GitHub has flagged my account because I apparently have registered free multiple user accounts?! I only created a GitHub account last year for a course I am doing. I have no idea why they think I have tried to create multiple accounts. I need to be able to share my account with my lecturer. What should I do?

**Comment by LunaBaloona** [comment]
Have the exact same problem 99% sure I don't have other accounts.  Did you ever get it solved with support?
